### ISSUE TITLE
Mikrotik routers MeshAgent

### ISSUE THREAD

**Issue by Raito00** [Ylianst/MeshCentral#5326]
There is support for: MeshAgent on MIPS OpenWRT.
So what about for MikroTik routers?

**Comment by si458** [comment]
mikrotik is a closed system/software, so how would you install the application?
i no, ive tried, we use there routers haha!
you can tho, install the agent at a remote site on a computer, then setup a relay group via that machine, then u can SSH, WebUI, or even use the meshcentralrouter to TCP portforward the winbox port!

**Comment by Raito00** [comment]
So one way is container in MikroTik ...

**Comment by si458** [comment]
> So one way is container in MikroTik ...
i through containers was only for ARM/ARM64  devices with mikrotik?
i dont use containers on our routers due to low memory

**Comment by PTR-inc** [comment]
Tried it on a vm with CHR where there is resource to spare. Works after quite some fiddling, but to get it to work on a physical device with limited resources is a challenge. I abandoned the idea as using a wireguard link to my management server is easier, builtin and more failsafe.
Nonetheless the container option on the mikrotik routers is an interesting add-on.
Or is there another use case to have a meshagent on a mikrotik router?
